packages/ui-patterns/form/FormItemLayout/FormItemLayout.stories.tsx

```diff
@@ -0,0 +1,297 @@
+  tags: ['autodocs'],
+  // More on argTypes: https://storybook.js.org/docs/api/argtypes
+  // argTypes: {},
+}
+
+type Story = StoryObj<typeof FormItemLayout>
+
+export const Primary: Story = {
+  render: function Render(args) {
+    const formSchema = z.object({
+      username: z.string().min(2, {
+        message: 'Username must be at least 2 characters.',
+      }),
+    })
+
+    // 1. Define your form.
+    const form = useForm<z.infer<typeof formSchema>>({
+      resolver: zodResolver(formSchema),
+      defaultValues: {
+        username: '',
+      },
+    })
+
+    // 2. Define a submit handler.
+    function onSubmit(values: z.infer<typeof formSchema>) {
+      // Do something with the form values.
+      // ✅ This will be type-safe and validated.
+      console.log(values)
+      // action('form form.handleSubmit(onSubmit)')(values)
+    }
+    return (
+      <Form_Shadcn_ {...form}>
+        <form className="w-96 flex flex-col gap-3" onSubmit={form.handleSubmit(onSubmit)}>
+          <FormField_Shadcn_
+            name="username"
+            control={form.control}
+            render={({ field }) => (
+              <FormItem_Shadcn_>
+                <FormItemLayout {...args}>
+                  <FormControl_Shadcn_>
+                    <Input placeholder="mildtomato" {...field} />
+                  </FormControl_Shadcn_>
+                </FormItemLayout>
+              </FormItem_Shadcn_>
+            )}
+          />
+          <Button size="small" type="primary" htmlType="submit">
+            Submit
+          </Button>
+        </form>
+      </Form_Shadcn_>
+    )
+  },
+  args: {
+    label: 'Username',
+    description: 'this is the description',
+    labelOptional: 'optional',
+    placeholder: 'shadcn',
+  },
+}
+
+export const withSelect: Story = {
+  render: function Render(args) {
+    const FormSchema = z.object({
+      email: z
+        .string({
+          required_error: 'Please select an email to display.',
+        })
+        .email(),
+    })
+
+    const form = useForm<z.infer<typeof FormSchema>>({
+      resolver: zodResolver(FormSchema),
+    })
+
+    // 2. Define a submit handler.
+    function onSubmit(values: z.infer<typeof FormSchema>) {
+      // Do something with the form values.
+      // ✅ This will be type-safe and validated.
+      console.log(values)
+      // action('form form.handleSubmit(onSubmit)')(values)
+    }
+
+    return (
+      <Form_Shadcn_ {...form}>
+        <form className="w-96 flex flex-col gap-3" onSubmit={form.handleSubmit(onSubmit)}>
+          <FormField_Shadcn_
+            name="email"
+            control={form.control}
+            render={({ field }) => (
+              <FormItemLayout {...args}>
+                <FormItem_Shadcn_>
+                  <FormControl_Shadcn_>
+                    <Select_Shadcn_ onValueChange={field.onChange} defaultValue={field.value}>
+                      <FormControl_Shadcn_>
```

Comment:
just to check, a bit confused here - why the repeat of FormControl here?

Reply:
just a mistake. funny it didn't break